certificatemanager/apiv1/certificate_manager_client_example_test.go

```diff
@@ -28,6 +28,11 @@ import (
 
 func ExampleNewClient() {
 	ctx := context.Background()
+	// This snippet has been automatically generated and should be regarded as a code template only.
```

Comment:
@quartzmo Maybe we should regen all gapics to get these doc updates out?

Reply:
OK, we can wait to merge this PR until after all gapics get the update.

Reply:
Does #6636 update everything?